LC_ALL=C for the error messages in the statement handle tests.

--- a/t/06attrs.t
+++ b/t/06attrs.t
@@ -3,6 +3,7 @@
 use strict;
 
 use Test::More tests => 145;
+use POSIX qw(setlocale LC_ALL);
 
 ## ----------------------------------------------------------------------------
 ## 06attrs.t - ...
@@ -17,6 +18,9 @@
 
 $|=1;
 
+# For the error messages in the statement handle tests.
+setlocale(LC_ALL, "C");
+
 my $using_autoproxy = ($ENV{DBI_AUTOPROXY});
 my $dsn = 'dbi:ExampleP:dummy';
 
